shared/actions/app.js

```diff
@@ -18,7 +18,7 @@ function mobileAppStateChanged(nextAppState: string): Constants.MobileAppState {
   return {payload: {nextAppState}, type: 'app:mobileAppState'}
 }
 
-function* _onMobileAppStateChanged(action: Constants.MobileAppState): SagaGenerator<any, any> {
+const _onMobileAppStateChanged = function*(action: Constants.MobileAppState): SagaGenerator<any, any> {
```

Comment:
this is the first of many examples of us having to change exported generators. It was caused by this really old bug that still exists
https://github.com/facebook/react-native/issues/14838
If we use this style vs just having them be the `function *` style then it works /shrug